pkg/controller/jobs/rayjob/rayjob_controller.go

```diff
@@ -175,6 +216,17 @@ func (j *RayJob) RestorePodSetsInfo(podSetsInfo []podset.PodSetInfo) bool {
 		info := podSetsInfo[index+1]
 		changed = podset.RestorePodSpec(&workerPod.ObjectMeta, &workerPod.Spec, info) || changed
 	}
+
+	// submitter
+	if j.Spec.SubmissionMode == rayv1.K8sJobMode {
+		submitterPod := j.Spec.SubmitterPodTemplate
+		if submitterPod == nil {
+			submitterPod = getDefaultSubmitterTemplate(&rayv1.RayCluster{Spec: *j.Spec.RayClusterSpec})
+		}
```

Comment:
I would like to commonize the logic of fallback to `getDefaultSubmitterTemplate` between the two uses.- maybe the `getSubmitterTemplate()` helper function?

Reply:
done